src/vm/dllimport.cpp

```diff
@@ -5775,8 +5792,9 @@ class LoadLibErrorTracker
         {
             SString hrString;
             GetHRMsg(theHRESULT, hrString);
-            COMPlusThrow(kDllNotFoundException, IDS_EE_NDIRECT_LOADLIB, libraryNameOrPath.GetUnicode(), hrString);
+            COMPlusThrow(kDllNotFoundException, IDS_EE_NDIRECT_LOADLIB_WIN, libraryNameOrPath.GetUnicode(), hrString);
         }
+#endif
```

Comment:
Nit, not worth resetting for, but normally after #else / #elif / #endif you add an inline comment noting what #if or #endif they match with. Eg., line 5785 would end with // __APPLE__ and 5797 with // FEATURE_PAL -- if I understand the convention correctly